pkg/kn/commands/service/describe.go

```diff
@@ -169,7 +169,10 @@ func writeService(dw printers.PrefixWriter, service *v1alpha1.Service) {
 		}
 	}
 	if (service.Spec.Template != nil) && (service.Spec.Template.Spec.ServiceAccountName != "") {
-		dw.WriteAttribute("ServiceAccount", service.Spec.Template.Spec.ServiceAccountName)
+		dw.WriteAttribute("Service Account", service.Spec.Template.Spec.ServiceAccountName)
+	}
+	if service.Spec.Template != nil && service.Spec.Template.Spec.ImagePullSecrets != nil {
+		dw.WriteAttribute("Image Pull Secrets", service.Spec.Template.Spec.ImagePullSecrets[0].Name)
```

Comment:
So you are using only the first element, right ? IMO it would be more consequent to call the option `--image-pull-secret` (singular) than `--image-pull-secrets`

Reply:
I removed `image` from it. Do you mean to add `image` in the flag as well apart from singular comment?

Yes, we're using only the first element.
IIUC only one image is allowed per Knative Service, but should we keep the flag name closer to the actual field name in spec?

Reply:
nope, `--pull-secret` is ok as the context is clear. My bad.